models/migrations/v16.go

```diff
@@ -19,9 +19,9 @@ type RepoUnit struct {
 	RepoID      int64 `xorm:"INDEX(s)"`
 	Type        int   `xorm:"INDEX(s)"`
 	Index       int
-	Config      map[string]string `xorm:"JSON"`
-	CreatedUnix int64             `xorm:"INDEX CREATED"`
-	Created     time.Time         `xorm:"-"`
+	Config      map[string]interface{} `xorm:"JSON"`
```

Comment:
Because it could not only be a string. It could also be a boolean like it is used in the latest migration.

Reply:
Or is there a good alternative to this? @bkcsoft

Reply:
I'd say "define an interface", but that would be just as hard to maintain... Overall I really don't like using generic configs like this... I'd just make a new bool flag for `EnableTimetracking` etc 🤔

Reply:
Alternative to that is to make `Config` a first class citizen, so it would be 
```go
Config RepoConfig `xorm:"JSON"`
```
But that would be out-of-scope for this PR 😄

Reply:
@bkcsoft I think the solution with having `Config` as a first class citizen is the best one but as you already mentioned it would be better to create a  new PR for this. I propose to leave it as it is as a temporary solution until the "new" PR is done.